src/qml/components/BlockClockComponent.qml

```diff
@@ -0,0 +1,64 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../controls"
+
+BlockClock {
+    id: blockClock
+    anchors.centerIn: parent
+    synced: nodeModel.verificationProgress > 0.999
+    pause: nodeModel.pause
+    // TODO: Modify following logic once nodeModel.numOutboundPeers property is incorporated.
+    conns: true
+
+    states: [
+        State {
+            name: "intialBlockDownload"; when: !synced && !pause && conns
+            PropertyChanges {
+                target: blockClock
+
+                ringProgress: nodeModel.verificationProgress
+                header: Math.round(ringProgress * 100) + "%"
+                subText: Math.round(nodeModel.remainingSyncTime/60000) > 0 ? Math.round(nodeModel.remainingSyncTime/60000) + "mins" : Math.round(nodeModel.remainingSyncTime/1000) + "secs"
+            }
+        },
+
+        State {
+            name: "blockClock"; when: synced && !pause && conns
+            PropertyChanges {
+                target: blockClock
+
+                ringProgress: blockList[0]
+                header: Math.floor(nodeModel.blockTipHeight / 1000) + "," + nodeModel.blockTipHeight % 1000
+                subText: "Blocktime"
+                blockList: chainModel.timeRatioList
```

Comment:
QML error appears when shutting down.

```
2022-12-17T03:52:37Z Shutdown: In progress...
2022-12-17T03:52:37Z torcontrol thread exit
2022-12-17T03:52:37Z addcon thread exit
2022-12-17T03:52:37Z msghand thread exit
2022-12-17T03:52:37Z net thread exit
2022-12-17T03:52:39Z opencon thread exit
2022-12-17T03:52:39Z DumpAnchors: Flush 0 outbound block-relay-only peer addresses to anchors.dat started
2022-12-17T03:52:39Z DumpAnchors: Flush 0 outbound block-relay-only peer addresses to anchors.dat completed (0.01s)
2022-12-17T03:52:39Z scheduler thread exit
2022-12-17T03:52:39Z Writing 0 unbroadcast transactions to disk.
2022-12-17T03:52:39Z Dumped mempool: 3.7e-06s to copy, 0.00448635s to dump
2022-12-17T03:52:39Z Shutdown: done
2022-12-17T03:52:39Z GUI: qrc:/qml/components/BlockClockComponent.qml:34: TypeError: Cannot read property 'timeRatioList' of null
```

Reply:
Addressed in bcbc80b6666e43aa07ee46e09289b4a34f3fa30b